package.json

```diff
@@ -26,8 +26,12 @@
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
     "react-styleguidist": "^6.2.7",
+    "markdown-to-jsx": "6.4.1",
```

Comment:
This doesn't have to appear after react-styleguidist of course... I was just hacking towards a solution